### DISCUSSION TITLE
How to configure retention period for so-playbook-alerts

### DISCUSSION THREAD

**Discussion by presianbg** [Security-Onion-Solutions/securityonion#7865] [Unsupported Versions] [answered]
Hi,

I'm still not able to configure a geolocation maps in Kibana for our alerts.
I've started from here https://github.com/Security-Onion-Solutions/securityonion/issues/3152#issuecomment-1069059886 to find why field `destination_geo.location` is still showing up when I try to configure a Map visualization:
![image](https://user-images.githubusercontent.com/2163486/165523432-a26ec946-0fbe-4073-9d06-ae4a8c4625d7.png)
I've checked and all of the indices on our search nodes are rolled over.

What I found is that the `so-playbook-alerts` index is stored on the manager node and its never closed / deleted:
![image](https://user-images.githubusercontent.com/2163486/165523755-57f87fb5-3315-49e5-8eec-55ab1c28c9cc.png)

I couldn't find where and how to configure a retention period for the alerts generated from `Playbooks` .

Is there someone having the same issue ?

Cheers,
PY

**Comment by weslambert** [comment, marked as answer]
For the time being you should be able to manually close indices with something like the following:

```
# Close indices for March and April

for i in $(so-elasticsearch-query _cat/indices | grep -E "so-playbook-alerts-2022.03*|so-playbook-alerts-2022.04*" | awk '{ print $3}'); do so-elasticsearch-query $i/_close -XPOST ; done
```

I wouldn't be too concerned with disk space, since the Playbook alert index size should not be too large. I'll look into the current scheme, if any, around maintenance.

**Reply by presianbg** [reply]
Thank you, I will try that.
My concern is regarding conflicting geo special fields, just noticed that I have playbook alerts since first SO 2 installation.
